src/platform/icelake/include/platform/clk-map.h

```diff
@@ -31,14 +31,19 @@
 #ifndef __INCLUDE_CLOCK_MAP__
 #define __INCLUDE_CLOCK_MAP__
 
+#include <sof/clk.h>
+
 static const struct freq_table cpu_freq[] = {
 	{120000000, 120000, 0x0},
 	{400000000, 400000, 0x4}, /* default */
 };
 
+/* IMPORTANT: array should be filled in increasing order
+ * (regarding to .freq field)
+ */
 static const struct freq_table ssp_freq[] = {
-	{19200000, 19200, }, /* default */
-	{38400000, 38400, },
+	{ 38400000, 38400, CLOCK_SSP_XTAL_OSCILLATOR },
+	{ 96000000, 96000, CLOCK_SSP_PLL_FIXED },
```

Comment:
please double check, I was told by hardware folks that the 24.576 MHz PLL was available on ICL.

Reply:
I've checked ICL configuration with Audio Cardinal Clock and it works. I've have updated ssp_freq array for ICL. Thanks.